awx_collection/test/awx/test_build_url.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+from unittest import mock
+
+__metaclass__ = type
+
+import pytest
+
+
+@pytest.mark.parametrize(
```

Comment:
This looks like a matrix of repeated things. It is possible to produce it using several stacked parametrize decorators.